components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -58,19 +58,27 @@ class BrowserToolbar @JvmOverloads constructor(
     /**
      * Set/Get whether a site security icon (usually a lock or globe icon) should be next to the URL.
      */
-    var displaySiteSecurityIcon: Boolean
+    var siteSecurityIcon: Boolean
         get() = displayToolbar.siteSecurityIconView.isVisible()
         set(value) {
             displayToolbar.siteSecurityIconView.visibility = if (value) View.VISIBLE else View.GONE
         }
 
+    var siteSecurityColor: Pair<Int, Int>
```

Comment:
We can improve the name of the variable because it is not clearly conveying its purpose.
As it is a pair it will contain two colours but I'm not quite sure which state is which. Maybe adding a comment?

Reply:
Added comments!